pusher/src/Controller/IoSocketController.ts

```diff
@@ -177,7 +178,8 @@ export class IoSocketController {
                         }
 
                         const userUuid = await jwtTokenManager.getUserUuidFromToken(token);
```

Comment:
`getUserUuidFromToken` is calling `adminApi.fetchCheckUserByToken` behind the scene.
We could probably use this call to verify whether the user is banned or not.
We would save one round trip to the admin server.

Reply:
Yes, you have right, I will change this code.